src/ui/units/wizard/components/ValuesList/ValuesList.tsx

```diff
@@ -202,22 +203,26 @@ class ValuesList extends React.Component<Props, State> {
     };
 
     async fetchInitialData() {
-        const {item, distincts: externalDistincts} = this.props;
+        const {item, distincts: externalDistincts, items} = this.props;
         const {isRetry} = this.state;
 
         this.setState({
             fetching: true,
         });
 
         try {
-            let distincts;
             let values: string[] = [];
-
             if (externalDistincts) {
-                values = externalDistincts[item.guid];
+                const placeholderFields =
+                    this.props.isMultipleFieldsSupported && items ? items : [item];
+
+                const distincts = placeholderFields.map((v) => {
+                    return externalDistincts[v.guid || v.title];
```

Comment:
I'm not sure if you can rely on the title here. Especially for multiple fields

Reply:
yes, guid is more than enough in that case